src/stable_browser.ts

```
@@ -2404,49 +2404,55 @@ class StableBrowser {
       while (true) {
-        const resultWithElementsFound = await this.findTextInAllFrames(
-          dateAlternatives,
-          numberAlternatives,
-          text,
-          state
-        );
+        const resultWithElementsFound = {
+          length: 0,
+        };
+        try {
+          resultWithElementsFound = await this.findTextInAllFrames(dateAlternatives, numberAlternatives, text, state);
```

Biome (1.9.4)

[error] Can't assign resultWithElementsFound because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)